src/bmipy/cmd.py

```diff
@@ -1,22 +1,31 @@
 """Command line interface that create template BMI implementations."""
+from __future__ import annotations
+
+import argparse
+import functools
 import inspect
 import keyword
 import re
+import sys
 
-import black as blk
-import click
 import jinja2
+from bmipy._version import __version__
+from bmipy.bmi import Bmi
 
-from bmipy import Bmi
+try:
+    import black as blk
+except ModuleNotFoundError:
+    WITH_BLACK = False
+else:
+    WITH_BLACK = True
 
-BMI_TEMPLATE = """# -*- coding: utf-8 -*-
-{% if with_hints -%}
-from typing import Tuple
-{%- endif %}
+BMI_TEMPLATE = """\
+from __future__ import annotations
 
-from bmipy import Bmi
 import numpy
```

Comment:
In the output from `bmipy-render`, *numpy* is referred to as *np* in several signatures.

Reply:
I've included `import numpy as np` to satisfy the linters but I don't think it's needed to run the code. Because we import *annotations* from the future, the type annotations are stringified before the code is executed to enable delayed evaluation.

In the future, we may want to annotate the *numpy* types with strings to remove the *numpy* requirement.